tests/test_self.py

```diff
@@ -103,4 +110,34 @@ def test_gw(client):
 
     assert r.json['mjds']['gwproxy.GWProxyResolver'] is not None
     assert r.json['success'] == True
-    assert r.json['success_time'] == True
\ No newline at end of file
+    assert r.json['success_time'] == True
+
+
+@pytest.mark.parametrize('source_name', ['Crab', 'Mrk 421', 'aaaaaa'])
+def test_byname_11(resolver_live_fixture, source_name):
```

Comment:
hm, why cannt you use `client` here like in the other tests?

Reply:
simply I didn't manage to make it work and didn't spend too much time investigating, and moved on to something I was more confident with...

Reply:
what's the issue?

the reason why it's different than dispatcher is that dispatcher needs full live server due its various complexities. But `client` is much faster and does not have deal with starting another process.

do none of the tests work with `client`? or only this one?

Reply:
ok after some more investigation I managed to make it work, basically it ws an issue in my virtualenv, and I had to install some missing libraries.

I will update the test I wrote, and I agree, it will be way more easy to handle

Reply:
using now only `client` fixture, removed unused ones